drivers/rail/include/pa_conversions_efr32.h

```diff
@@ -0,0 +1,82 @@
+/***************************************************************************//**
```

Comment:
Is this something I should add to `gecko_sdk`?

Reply:
yeah that would be better, I just copied it from the sdk (/gecko_sdk_suite/v2.2/platform/radio/rail_lib/plugin).

Reply:
ok, I changed the PKG_VERSION of the gecko_sdk to the branch with the new version and removed the files from the driver (and dealt with some API changes ...)